script.js

```diff
@@ -0,0 +1,42 @@
+var grid = document.querySelector("#grid");
+var whichPlayer = "X"
+var numMoves = 0
+grid.addEventListener("click", markSquare);
+
+
+
+function markSquare(square) {
+  if (square.target != square.currentTarget) {
+    var clicked = square.target.id;
+    playerMove(clicked);
+  }
+}
+
+function playerMove(clicked) {
+    if (document.getElementById(clicked).innerHTML !== ""){
+      alert("That square is already marked");
+    }
```

Comment:
i think you could just do `if (document.getElementByID(clicked).innerHTML)`

utilize truthy and falsey to your advantage